src/install.rst

```diff
@@ -73,9 +73,7 @@ These instructions will install the Nextstrain CLI and tools to run and view you
                1. Install Docker Desktop using `the official guide <https://docs.docker.com/desktop/mac/install/>`_.
                2. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-                  .. code-block:: bash
-
-                     mamba create -n nextstrain -c conda-forge -c bioconda nextstrain-cli --yes
+                  .. include:: snippets/conda-install-minimal.rst
 
```

Comment:
Add something along the lines of

```suggestion
conda config --env --set channel_priority strict
conda config --env --add channels conda-forge
conda config --env --add channels bioconda
```

to have the environment set up once and for all and being able to update without having to add flags.

Otherwise things will break if people update and forget a flag. We could say "optional" here, but I would really like this to be mentioned in the docs. It's just 1 extra copy/paste and adds a lot of robustness.

Reply:
In fact we even suggest this type of *future-proofing* for apple silicon, so I don't see this as breaking with convention, and convention shouldn't prevent us from encouraging robustness:
>  Ensure future Conda commands in this environment use Intel packages too.
> ```conda config --env --set subdir osx-64```

Reply:
I would add this as a snippet so it can be reused in many places (as it should be, whenever one sets up an environment this should be run - if not included in ~/.condarc in general)

Reply:
I'm also in favor of adding this since:

1. We have the code block copy button, so it's not much extra work on the user.
2. It's only an extra step for the native runtime setup, doesn't affect Docker runtime.
3. I'd expect long-term users of the native runtime to eventually want to use this config either in the env or `~/.condarc`.

Reply:
Sure, that's fine. I was trying to avoid a larger rework of these install commands/instructions, but I agree it's more robust to set in config and agree that it's likely desirable long-term for users.

That said, we use `config config --env` for the M1 users instead of modifying the global config. Should we do that for channels too so we don't modify their global config?

I repushed just now to add the command-line options to the `mamba update` invocation, which was missed, but this will go away with the use of config.

Reply:
Sounds great! Thanks for thorough checking and thinking through!

Reply:
Done in dc1b9a5.